spec/furniture/journal/keyword_spec.rb

```diff
@@ -7,6 +7,24 @@
   it { is_expected.to validate_uniqueness_of(:canonical_keyword).case_insensitive.scoped_to(:journal_id) }
   it { is_expected.to belong_to(:journal).inverse_of(:keywords) }
 
+  describe "#merge" do
+    it "adds aliases for the other `Keywords` canonical and aliases, deletes the other Keyword, and re-detects the entries keywords" do
```

Comment:
This might be overly paranoid, but I wonder if it would be worth adding an assertion that if the `from` keyword is associated to any entries, those don't get deleted when the keyword is destroyed.

Reply:
Not too paranoid!